mtags/src/main/scala/scala/meta/internal/pc/CompletionProvider.scala

```diff
@@ -369,9 +369,22 @@ class CompletionProvider(
       val isTypeMember = kind == CompletionListKind.Type
       params.checkCanceled()
       val matchingResults = completions.matchingResults { entered => name =>
-        if (isTypeMember) CompletionFuzzy.matchesSubCharacters(entered, name)
-        else CompletionFuzzy.matches(entered, name)
+        /** NOTE(tgodzik): presentation compiler bug
+         *  in case we have a comment inline presentation compiler will see it as the name
+         *  CompletionIssueSuite.issue-813 for more details
+         */
+        val enteredFixed = if (entered.startsWith("$div$div")) {
```

Comment:
Could you add comments to make those "$div$div" more readable? like:
- `// start of a comment: "//"`
- `// entire one-line comment`